pyquil/tests/test_paulis.py

```diff
@@ -270,9 +270,9 @@ def test_zero_terms():
     assert str(term4) == "0j*I"
 
     term = PauliSum([PauliTerm("X", 0, 0.0), PauliTerm("Y", 1, 1.0) * PauliTerm("Z", 2)])
-    assert str(term) == "0j*X0 + (1+0j)*Y1*Z2"
```

Comment:
how was this ever passing?

Reply:
Uh? `str` joined on "*" so I expected `Y1*Z2`